tests/unittest_regrtest.py

```diff
@@ -156,6 +159,38 @@ class B(compiler.__class__):
         base = next(result._proxied.bases[0].infer())
         self.assertEqual(base.name, "int")
 
+    @pytest.mark.skipif(not PY38_PLUS, reason="needs assignment expressions")
```

Comment:
Congrats ! First try this time !

Reply:
I guess you didn't see the third commit 😄